.circleci/config.yml

```diff
@@ -0,0 +1,60 @@
+version: 2.1
+
+jobs:
+    build:
+        docker:
+            -   image: akeneo/php:7.2
+        steps:
+            -   checkout
+            -   run: composer install
+            -   persist_to_workspace:
+                    root: ~/
+                    paths:
+                        - project
+
+    test_php:
+        docker:
+            -   image: akeneo/php:7.2
+        steps:
+            -   attach_workspace:
+                    at: ~/
+            -   run: bin/phpunit -c phpunit.xml.dist
+            -   run: bin/phpspec run
+
+    test_php_code_style:
+        docker:
+            -   image: akeneo/php:7.2
+        steps:
+            -   attach_workspace:
+                    at: ~/
+            -   run: bin/php-cs-fixer fix --diff --dry-run --config=.php_cs.php -vvv
+
+workflows:
+    pull_request:
+        jobs:
+            -   wait_for_user_approval:
+                    type: approval
+                    filters:
+                        branches:
+                            ignore:
+                                - master
+            -   build:
+                    requires:
+                        - wait_for_user_approval
+            -   test_php:
+                    requires:
+                        - build
+            -   test_php_code_style:
+                    requires:
+                        - build
```

Comment:
add a `workflow_success` as a last step in order to configure github on this specific step to allow merge when green.

something like:


```
- workflow_success:
    requires:
        - test_php
        - test_php_code_style
```

Reply:
Added.  
Is this different from the current behavior or is this for future purpose?